lib/Bio/EnsEMBL/DataCheck/Checks/CheckOntologyTerm.pm

```diff
@@ -0,0 +1,54 @@
+=head1 LICENSE
+
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckOntologyTerm;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckOntologyTerm',
+  DESCRIPTION    => 'Check presence of critical terms',
+  GROUPS         => ['ontologies'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['ontology'],
+  TABLES         => ['term'],
+};
+
+sub tests {
+  my ($self) = @_;
+
+  my $desc = 'Critical terms are present in '.$self->dba->dbc->dbname;
```

Comment:
It doesn't matter a great deal, but note that the database name is included automatically in the output, so you don't need it in the description, e.g. it'll be something like:
```
# Subtest: CheckOntologyTerm
    # Subtest: multi, ontology, ensembl_ontology
        ok 1 - Critical terms are present in ensembl_ontology
```